Dockerfile

```diff
@@ -1,12 +1,11 @@
 FROM alpine:3.10
 
 ARG ARCH=
-ARG VERSION="v1.17.2+k3s1"
 
-ENV K3S_RELEASE https://github.com/rancher/k3s/releases/download/${VERSION}/k3s${ARCH}
+ENV K3S_RELEASE https://github.com/rancher/k3s/releases/download/${DRONE_TAG}/k3s${ARCH}
```

Comment:
Is `${DRONE_TAG}` available when invoking `./scripts/package` locally? Even if so, leaving it as `${VERSION}` would be less confusing.